internal/pkg/template/templates/addons/s3/cf.yml

```diff
@@ -19,7 +19,6 @@ Resources:
         ServerSideEncryptionConfiguration:
         - ServerSideEncryptionByDefault:
             SSEAlgorithm: AES256
-      BucketName: !Sub '${App}-${Env}-${Name}-{{.Name}}'
```

Comment:
nit: can we also update the logical name from `{{logicalIDSafe .Name}}` to `{{logicalIDSafe .Name}}Bucket`